### ISSUE TITLE
Build error

### ISSUE THREAD

**Issue by jrazer** [popcorntime/popcorntime#352]
Operating System Version: 
Ubuntu 16.10

Popcorn Time Version: 
0.3.10

Download date: 
20.11.2016

Error on npm install: 
npm WARN notsup Not compatible with your operating system or architecture: fsevents@1.0.15
npm ERR! Linux 4.8.0-27-generic
npm ERR! argv "/usr/bin/nodejs" "/usr/bin/npm" "install"
npm ERR! node v4.2.6
npm ERR! npm  v3.5.2
npm ERR! code ELIFECYCLE

npm ERR! guppy-pre-commit@0.4.0 postinstall: `guppy pre-commit`
npm ERR! Exit status 1
npm ERR! 
npm ERR! Failed at the guppy-pre-commit@0.4.0 postinstall script 'guppy pre-commit'.

#### Steps to reproduce the behaviour
1. sudo npm install -g bower gulp
2. npm install

**Comment by yassiezar** [comment]
I got the same error, but I managed to fix it. 

Before the npm error output, there was a line 

    sh: node: command not found
It seems that installing nodejs didn't create a symlink to node on my system (Mint 18). If you're using Ubuntu 16 you might have the same issue. Check if `node` is a valid command, otherwise issue 

    sudo ln -s `which nodejs` /usr/bin/node
to manually create the correct symlink and it should be fixed. Popcorntime is building successfully for me now.
